### ISSUE TITLE
How to Calculate Metrics

### ISSUE THREAD

**Issue by Xiao0219** [song-wensong/insert-anything#39]
Hello, I'd like to ask how you calculate metrics. Are the restored results directly compared with the target image to calculate metrics? But some images seem unreasonable. For example，Should the result image be used to calculate metrics against the source image?

<img width="1228" height="231" alt="Image" src="https://github.com/user-attachments/assets/b0def05c-0b3e-4ae1-89bf-59b2b819ca2f" />

**Comment by song-wensong** [comment]
Hello, the images in your screenshot are qualitative comparison figures. In our paper, the quantitative metrics for person insertion are calculated using the test-person-simple_scene, which has target images (ground truth). The test-person-complex_scene dataset does not include ground truth; it only contains reference images and source images. These are images we collected from the internet to further test generalization, and they are presented in the qualitative comparisons.

**Comment by Xiao0219** [comment]
> Hello, the images in your screenshot are qualitative comparison figures. In our paper, the quantitative metrics for person insertion are calculated using the test-person-simple_scene, which has target images (ground truth). The test-person-complex_scene dataset does not include ground truth; it only contains reference images and source images. These are images we collected from the internet to further test generalization, and they are presented in the qualitative comparisons.

Hi, thank you for your reply. I would like to confirm: the test dataset you provided on Hugging Face seems to be only for qualitative comparison, is that correct?
If so, then where can I find the dataset used for quantitative evaluation in your paper?

I read your paper carefully, and in the quantitative experiments, you mentioned the AnyInsertion datasets for person, object, and clothes insertion. This makes me a bit confused — does it mean that the datasets used for quantitative evaluation have not been released publicly? Or did I misunderstand the description in the paper?

I’m a bit unsure about this, and I hope you could help clarify it for me. Thank you!

**Comment by Xiao0219** [comment]
> The test set is placed at the following link: https://drive.google.com/drive/folders/1uUD_iGm4dAEVYGmXp5KvjWl1e_j2pjiK?usp=sharing. You can find the datasets used for quantitative evaluation there.
> 
> The test images for person insertion only use the person/simple_scene from the above link.

Here's the English translation:

The mask prompt downloaded from this link: [https://huggingface.co/datasets/WensongSong/AnyInsertion_V1/tree/main](https://www.google.com/search?q=https://huggingface.co/datasets/WensongSong/AnyInsertion_V1/tree/main) is for the training set and qualitative test set. This link: [https://drive.google.com/drive/folders/1uUD_iGm4dAEVYGmXp5KvjWl1e_j2pjiK?usp=sharing](https://www.google.com/search?q=https://drive.google.com/drive/folders/1uUD_iGm4dAEVYGmXp5KvjWl1e_j2pjiK%3Fusp%3Dsharing) is for the quantitative test set.

I finally figured it out. Thank you very much for your reply, and I wish your paper gets accepted!

**Comment by Xiao0219** [comment]
> The test set is placed at the following link: https://drive.google.com/drive/folders/1uUD_iGm4dAEVYGmXp5KvjWl1e_j2pjiK?usp=sharing. You can find the datasets used for quantitative evaluation there.
> 
> The test images for person insertion only use the person/simple_scene from the above link.

The mask prompt downloaded from this link: [https://huggingface.co/datasets/WensongSong/AnyInsertion_V1/tree/main](https://www.google.com/search?q=https://huggingface.co/datasets/WensongSong/AnyInsertion_V1/tree/main) is for the training set and qualitative test set. This link: [https://drive.google.com/drive/folders/1uUD_iGm4dAEVYGmXp5KvjWl1e_j2pjiK?usp=sharing](https://www.google.com/search?q=https://drive.google.com/drive/folders/1uUD_iGm4dAEVYGmXp5KvjWl1e_j2pjiK%3Fusp%3Dsharing) is for the quantitative test set.

I finally figured it out. Thank you very much for your reply, and I wish your paper gets accepted!
